data/paper.org

```diff
@@ -129,10 +129,11 @@ initial upvotes.
 #+ATTR_LATEX: :placement [H]
 [[./paper_figure_2.png]]
 
-If you invest in a meme that has 500 initial upvotes, it needs to earn approximately 60
+If you invest in a meme that has 500 initial upvotes, it needs to earn approximately 324
 additional upvotes before maturity for you to break even, but the loss if you
 fail to do so is less than with the penny stock. If the meme gains approximately
-190 upvotes, you'll double your money.
+500 upvotes or more, the return will be \sim115%. Beyond this, the return caps out at \sim123%
```

Comment:
Should be 15% and 23%, instead of 115% and 123%

Reply:
:+1:

Reply:
@rickles42 I'm not totally sure now. For eg, on the lower limit, we say the return is 5%, i.e. you get back 5% of what you put in. In a similar way, here, the return is 115% and 123% and the *profit* is 15% and 23%

Reply:
It's all question of how we define it really.

Reply:
Sorry for the confusing phrasing. These should definitely be 15% and 23% profit or gain. The 5% is confusing because it's also a profit as well. Look at the curve, it goes up and maxes out between 1.0 and 1.5, representing a gain of 0%-50%. Feel free to improve the phrasing around the 5% number if you want.

Reply:
Ok I think the best way is to say 15% and 23% profit. and then 95% loss for the lower limit. I'll reword it accordingly.

Reply:
I don't think that number is talking about a lower limit to loss. It's supposed to be talking about the upper limit of gains, but for very large-cap memes. Does that make sense? The number should be 20% like it was before.